### DISCUSSION TITLE
import custom routing xml

### DISCUSSION THREAD

**Discussion by fixifoxy** [osmandapp/OsmAnd#14679] [General]
I´ve got 3 custom routing profiles in file format .xml.
In the past I  created a new profile based on, 
let say walking profile,  
find the profile folder in Osmands routing folder,
 renamed it  
and copied the custom routing-xml into the folder of the new profile.
Now no  file/folder is shown in the routing folder, even when I create a new profile.
How can I import my custom routing files to use it in profile?

When following the offered Osmand procedure and selecting the xml file under "settings" "import profile" the xml-Format is not accepted. It is asked for OSF-format.

### Your Environment
OsmAnd Version:4.2.5
Android/iOS version:12
Device model:Samsung S21

**Maps used (online or offline):**   
If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs and its edition date.

**Comment by vshcherb** [comment]
Here were more details included https://github.com/osmandapp/OsmAnd/issues/14691, but it's still not clear whether it's a bug or how it was imported. It's probably that OSF file was not constructed correctly, so in the end you could attach an example so we could test it.
